### PR TITLE
Search patient

### PR DESCRIPTION
**Pull request by ngarivictor**

### CODE REVIEW

File: omod/pom.xml [hunk 1]
```diff
@@ -113,6 +113,26 @@
 			<groupId>org.openmrs.module</groupId>
 			<artifactId>referenceapplication-omod</artifactId>
 		</dependency>
+        <dependency>
+            <groupId>org.openmrs.module</groupId>
+            <artifactId>kenyaemr-api</artifactId>
+            <scope>compile</scope>
```
Comment (ningosi) [inline omod/pom.xml]: please revise compile to provided

Comment (ningosi) [reply omod/pom.xml]: those might have been fixed in the sub sequent commits

Comment (ngarivictor) [reply omod/pom.xml:119]: okay

Comment (ngarivictor) [reply omod/pom.xml]: Done

**Review by ningosi — DISMISSED** [review]
Thank you @ngarivictor if only everyone would be this versatile ...

File: omod/pom.xml [hunk 2]
```diff
@@ -121,19 +121,8 @@
 		<dependency>
 			<groupId>org.openmrs.module</groupId>
 			<artifactId>kenyaemr-api</artifactId>
-			<scope>compile</scope>
+			<scope>provided</scope>
```
Comment (ningosi) [inline omod/pom.xml]: once if is set in the main pom.xml, might NOT be needed here, only groupId and artifactId suffice

Comment (ngarivictor) [reply omod/pom.xml]: Aaahh I see..

Comment (ngarivictor) [reply omod/pom.xml]: should be good now

**Review by ningosi — APPROVED** [review]

**Pull request merged**